### ISSUE TITLE
Person Skill View: Allow adding of Skills

### ISSUE THREAD

**Issue by Robin481** [puzzle/skills#639]
After implementing the Person Skill view here: https://github.com/puzzle/skills/issues/608 we want to continue with completing the Person Skill view.
Therefore, the ability to add a new but already existing Skill to the profile should be possible.
Furthermore, the ability to create a new Skill and then also add it to the profile should be possible. 

Example from the old application for adding a new but already existing Skill to the profile:
![image](https://github.com/puzzle/skills/assets/32168247/a47c6a0e-aa47-44bb-8926-e1fa34c0bc0b)

Example from the old application for creating and adding a new Skill:
![image](https://github.com/puzzle/skills/assets/32168247/8b2687ea-84ab-4367-9ea3-6586baaf8b7f)
 
TODO:
- Add functionality for adding both existing skills to people_skills and creating new ones... and then adding them to the profile as people_skill :grin: 
- If we can save a lot of time by creating skills only in a more central place instead of ad-hoc we can discuss that
- Allow adding of all relevant fields when adding a people_skill (competence_level, interest_level, core_competence, certificate)
- Don't allow the user to set anything relevant on the Skill itself such as default-set etc. This all only happens in the Skillset by Admins
-  Basically I could write a lot more but just be reasonable and orient yourself on the old application.
- Tests
- :rock: 
- :rocket:
